modules/cpmso-openstack-ts-az-config.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control_plane_machine_management/cpmso-troubleshooting.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-openstack-ts-az-config_{context}"]
+= Configuring {rh-openstack} Availability Zones type
```

Comment:
No need to capitalize "availability zones."

Reply:
Ack.